server/asks.ts

```diff
@@ -1,4 +1,11 @@
-const asks = {
+interface Asks{
```

Comment:
again this should not be plural - it describes a single ask only

Reply:
noted